### PR TITLE
CURA-8889_fix_one_at_a_time_initial_temperature

### PR DESCRIPTION
**Pull request by wawanbreton**
Bed temperature was not applied after the first model being printed.
Only for Griffin-flavored GCode printers (which includes UltiMaker printers).

### CODE REVIEW

**Review by rburema — APPROVED** [review]
Looks good to me, straightforward change.

**Comment by wawanbreton** [comment]
Thanks,
I tried to investigate the benchmark crash, and it also happens on the main branch. Should we wait for a fix before we can merge this one ?

**Comment by wawanbreton** [comment]
Ok, I have my answer :)

**Comment by rburema** [comment]
@wawanbreton Haha, within the same minute, even. The same, unrelated, benchmark crash is happening for a lot of branches. We should fix that soon, but since this isn't something I'd expect any changes in the benchmark from, we can just ignore it for now.

**Pull request merged**
